src/motor_instructions.py

```diff
@@ -1,11 +1,23 @@
 #!/usr/bin/python2.7
-
+from pickle import GLOBAL
+import RPi.GPIO as GPIO 
 import rospy
 from std_msgs.msg import Empty, Float32
 from CameraDataManager import CameraDataManager
-import SystemState
 
 
+INPUT_PIN = 18 # pin 12 on the real board
+
+def pin_setup():
+    # Pin Setup:
+    GPIO.setmode(GPIO.BCM)  # BCM pin-numbering scheme from Raspberry Pi
+    GPIO.setup(INPUT_PIN, GPIO.IN, pull_up_down=GPIO.PUD_UP)  # set pin as an input pin
+
+def read_pin():
+    global PREV_VALUE
```

Comment:
I might be missing what this is doing but this doesn't seem to be referenced.

Reply:
this is dead code see all commits instead of just this one.